src/vonage/client.py

```diff
@@ -134,10 +127,10 @@ def __init__(
         self.application_v2 = ApplicationV2(api_server)
         
         self.account = Account(self)
-        self.message_search = MessageSearch(self)
         self.messages = Messages(self)
         self.number_insight = NumberInsight(self)
         self.numbers = Numbers(self)
+        self.redact = Redact(self)
```

Comment:
Just flagging that if we decide to take Redact out we need to remove it from here as well